### DISCUSSION TITLE
Is htmx actively maintained/ improved?

### DISCUSSION THREAD

**Discussion by jamilraichouni** [bigskysoftware/htmx#3060] [General]
Hi!

I am asking because I just saw that the last commit on the htmx-2.0 branch was in July 2023.
htmx is a core technology needed for FastHTML to work and I ask myself if I use something that will die somewhen.

Hope you'll keep up the cool work!

Jamil

**Comment by Telroshan** [comment]
Hey, htmx 2 has already [been released](https://htmx.org/posts/2024-06-17-htmx-2-0-0-is-released/), and is now at the `master` branch!
It is indeed actively maintained, development occurs on the `dev` branch
At some point we'll have to clean up those branches a bit 😆

**Comment by jamilraichouni** [comment]
Thank you!

**Comment by nickchomey** [comment]
FYI, each repo has a "releases" page that you can check for the latest version, rather than look at branches. 

Here's HTMX's https://github.com/bigskysoftware/htmx/releases

**Comment by jamilraichouni** [comment]
Yes, thanks! I am actively developing on GitHub. Was just checking commits and so that branch.
My fault 😀
